terraform/README.md

```diff
@@ -0,0 +1,57 @@
+# Terraform
+
+This directory focuses on **terraforming** all the AWS cloud services that are used throughout the Railway Tracker application.
+
+
+## Installation
+
+1. Navigate to the project directory:
+
+```bash
+cd terraform
+```
+2. Install the required Python packages:
+
+```bash
+pip install -r requirements.txt
+```
```

Comment:
Do we even have a requirements.txt in the terraform folder?

Reply:
yes I agree, I will make the changes